assets/src/edit-story/components/animationTimeline/index.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import propTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import {
+  TimelineContainer,
+  TimelineContent,
+  TimelineLegend,
+  TimelineTimingContainer,
+  TimelineTitleBar,
+  TimelineRow,
+} from './components';
+
+export default function AnimationTimeline({ animations }) {
+  return (
+    <TimelineContainer>
+      <TimelineTitleBar>
+        <TimelineLegend />
+      </TimelineTitleBar>
+      <TimelineContent>
+        <TimelineLegend>
+          {animations.map((animation, index) => (
+            <TimelineRow
+              key={`timeline-animation-item-${animation.id}-legend`}
```

Comment:
Animations don't currently have an `id` property. Maybe something we want in the future tho 🤷 . Cool with it here for now since this PR is mostly about getting markup and base styles in, just wanted to give you a heads up that it would be `undefined` if you tried to throw in animations from an existing story.

I could see an `id` being useful/necessary tho as we get into more intense editing of the animation entries. can't remember if Mariano ran into this at all doing the story-anim-tool timeline.